reference/forms/types/options/help.rst.inc

```diff
@@ -0,0 +1,10 @@
+help
+~~~~
+
+**type**: ``string`` **default**: null
+
+Allows you to define a help text below the form field.
```

Comment:
What about something like: "Allows you to define extra information about the field, which will be rendered below the form field by default." (as you're allowed to change the layout of a form in every way you want)